jupyterhub/values.yaml

```diff
@@ -109,6 +109,16 @@ hub:
     minAvailable: 1
   networkPolicy:
     enabled: true
+    # To open for vmagent to scrape metrics from the hub pod, add the following
+    # to the ingress list:
+    # ingress:
+    # - from:
+    #   - namespaceSelector:
+    #       matchLabels:
+    #         name: victoriametrics
+    #   - podSelector:
+    #       matchLabels:
+    #         app.kubernetes.io/instance: vmagent
```

Comment:
I suggest removing these from here, as it's hard to maintain the info in two places and the documentation is the source of truth already.

```suggestion
```